src/Resources/views/CRUD/base_edit_form.html.twig

```diff
@@ -42,7 +42,12 @@
                                     {% set _tab_name = tab_prefix ~ '_' ~ loop.index %}
                                     <li{% if (not app.request.query.has('_tab') and loop.index == 1) or (tab_query_index == loop.index) %} class="active"{% endif %}>
                                         <a href="#{{ _tab_name }}" class="changer-tab" aria-controls="{{ _tab_name }}" data-toggle="tab">
-                                            <i class="fa fa-exclamation-circle has-errors hide" aria-hidden="true"></i> {{ form_tab.label|trans({}, form_tab.translation_domain ?: admin.translationDomain) }}
+                                            <i class="fa fa-exclamation-circle has-errors hide" aria-hidden="true"></i>
+                                            {% if form_tab.translation_domain is same as(false) %}
+                                                {{ form_tab.label }}
+                                            {% else %}
+                                                {{ form_tab.label|trans({}, form_tab.translation_domain ?: admin.translationDomain) }}
```

Comment:
`form_tab.translation_domain ?? admin.translationDomain`

Does twig support the ?? operator?

Reply:
Also on some places you use the ?:, on others ??, and others |default

IMO we should pick one.

Reply:
I tried to use ?? everywhere